tc/sgx/common/packages/base64/base64.h

```diff
@@ -9,12 +9,14 @@
 
 #pragma once
 
-#include "types.h"
+#include <string>
+#include <vector>
+//#include "types.h"
 
 #define BASE64_SIZE(x) (static_cast<size_t>(((((x) - 1) / 3) * 4 + 4) + 1))
```

Comment:
I do not see this used anywhere.
Also it will not work with types.h commented out.
I would delete it.
If you want to keep it, add a comment about what it does (length of a base 64 encoded string for a plaintext string? Including or excluding trailing NULL character?)

Reply:
I will delete BASE64_SIZE